src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -28,6 +28,8 @@ curie_map:
   ICD10WHO__2: http://apps.who.int/classifications/icd10/browse/2010/en#/
   OMIMPS: https://omim.org/phenotypicSeries/PS
   MEDGEN: http://identifiers.org/medgen/
+  MEDGEN2: http://identifiers.org/medgen/
```

Comment:
## Update to `mondo.sssom.config.yml`
I noticed that `mondo.sssom.config` was now not as up-to-date as `prefixes.csv`, so I updated it as well.

<details><summary>More info</summary>
<p>

When doing this, I noticed that actually there are more conflicts between these two files.. ugh, but I'm a bit fatigued from this now and I think maybe I'll leave it be and just hope to get to this one sooner:
- #146

</p>
</details>

Reply:
Just undid these changes as [I've just learned](https://github.com/monarch-initiative/mondo-ingest/pull/375#pullrequestreview-1741751315) that `mondo.sssom.config.yml` comes from `mondo`.